docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -40,68 +40,68 @@ configuration file will result in checkpoints being sorted by the loss metric in
      smaller_is_better: true
 
 The following snippet of Python code can be run after the specified experiment has generated a
-checkpoint. It returns an instance of :class:`~determined.experimental.client.Checkpoint`
-representing the checkpoint that has the best validation metric.
+checkpoint. It gets a list of sorted :class:`~determined.experimental.client.Checkpoint` instances
+associated with the experiment and selects the one with the best validation metric.
 
 .. code:: python
 
    from determined.experimental import client
 
-   checkpoint = client.get_experiment(id).top_checkpoint()
+   checkpoint = client.get_experiment(id).list_checkpoints()[0]
 
 Checkpoints can be sorted by any metric using the ``sort_by`` keyword argument, which defines which
-metric to use, and ``smaller_is_better``, which defines whether to sort the checkpoints in ascending
-or descending order with respect to the specified metric.
+metric to use, and ``order_by``, which defines whether to sort the checkpoints in ascending or
+descending order with respect to the specified metric.
```

Comment:
To sort checkpoints by any metric, use the ``sort_by`` argument to specify the metric and ``order_by`` to define the sorting order (ascending or descending).